page 20310
/bbs/Gossiping/M.1495115689.A.E19.html download to ptt/Gossiping/20310_0.txt, time: 5.550000
/bbs/Gossiping/M.1495115701.A.831.html download to ptt/Gossiping/20310_1.txt, time: 10.320000
/bbs/Gossiping/M.1495115731.A.ED4.html download to ptt/Gossiping/20310_2.txt, time: 5.430000
/bbs/Gossiping/M.1495115739.A.018.html download to ptt/Gossiping/20310_3.txt, time: 5.810000
/bbs/Gossiping/M.1495115801.A.3A6.html download to ptt/Gossiping/20310_4.txt, time: 4.450000
/bbs/Gossiping/M.1495115814.A.D06.html download to ptt/Gossiping/20310_5.txt, time: 5.780000
/bbs/Gossiping/M.1495115833.A.7D0.html download to ptt/Gossiping/20310_6.txt, time: 4.560000
/bbs/Gossiping/M.1495115925.A.476.html download to ptt/Gossiping/20310_7.txt, time: 7.030000
/bbs/Gossiping/M.1495115927.A.894.html download to ptt/Gossiping/20310_8.txt, time: 4.570000
/bbs/Gossiping/M.1495115947.A.D95.html download to ptt/Gossiping/20310_9.txt, time: 2.820000
/bbs/Gossiping/M.1495116023.A.A45.html download to ptt/Gossiping/20310_10.txt, time: 5.200000
/bbs/Gossiping/M.1495116062.A.ECE.html download to ptt/Gossiping/20310_11.txt, time: 4.870000
/bbs/Gossiping/M.1495116071.A.78A.html download to ptt/Gossiping/20310_12.txt, time: 5.640000
/bbs/Gossiping/M.1495116084.A.E12.html download to ptt/Gossiping/20310_13.txt, time: 5.300000
/bbs/Gossiping/M.1495116097.A.15D.html download to ptt/Gossiping/20310_14.txt, time: 6.580000
/bbs/Gossiping/M.1495116110.A.1DA.html download to ptt/Gossiping/20310_15.txt, time: 3.140000
/bbs/Gossiping/M.1495116112.A.CA7.html download to ptt/Gossiping/20310_16.txt, time: 4.460000
/bbs/Gossiping/M.1495116123.A.14D.html download to ptt/Gossiping/20310_17.txt, time: 3.540000
/bbs/Gossiping/M.1495116169.A.739.html download to ptt/Gossiping/20310_18.txt, time: 4.060000
/bbs/Gossiping/M.1495116195.A.6BB.html download to ptt/Gossiping/20310_19.txt, time: 5.850000
page 20309
/bbs/Gossiping/M.1495115180.A.E60.html download to ptt/Gossiping/20309_0.txt, time: 4.070000
/bbs/Gossiping/M.1495115231.A.BA3.html download to ptt/Gossiping/20309_1.txt, time: 5.950000
/bbs/Gossiping/M.1495115243.A.DC2.html download to ptt/Gossiping/20309_2.txt, time: 4.660000
/bbs/Gossiping/M.1495115248.A.49F.html download to ptt/Gossiping/20309_3.txt, time: 7.140000
/bbs/Gossiping/M.1495115268.A.024.html download to ptt/Gossiping/20309_4.txt, time: 5.470000
/bbs/Gossiping/M.1495115268.A.84A.html download to ptt/Gossiping/20309_5.txt, time: 3.470000
/bbs/Gossiping/M.1495115310.A.3E0.html download to ptt/Gossiping/20309_6.txt, time: 4.960000
/bbs/Gossiping/M.1495115345.A.922.html download to ptt/Gossiping/20309_7.txt, time: 5.140000
/bbs/Gossiping/M.1495115403.A.55A.html download to ptt/Gossiping/20309_8.txt, time: 4.290000
/bbs/Gossiping/M.1495115426.A.E66.html download to ptt/Gossiping/20309_9.txt, time: 3.870000
/bbs/Gossiping/M.1495115440.A.454.html download to ptt/Gossiping/20309_10.txt, time: 4.910000
/bbs/Gossiping/M.1495115472.A.4FD.html download to ptt/Gossiping/20309_11.txt, time: 4.440000
/bbs/Gossiping/M.1495115537.A.916.html download to ptt/Gossiping/20309_12.txt, time: 4.980000
/bbs/Gossiping/M.1495115541.A.152.html download to ptt/Gossiping/20309_13.txt, time: 5.030000
/bbs/Gossiping/M.1495115557.A.FF6.html download to ptt/Gossiping/20309_14.txt, time: 4.560000
/bbs/Gossiping/M.1495115558.A.993.html download to ptt/Gossiping/20309_15.txt, time: 5.550000
/bbs/Gossiping/M.1495115590.A.E8A.html download to ptt/Gossiping/20309_16.txt, time: 5.490000
/bbs/Gossiping/M.1495115617.A.EF2.html download to ptt/Gossiping/20309_17.txt, time: 7.800000
/bbs/Gossiping/M.1495115622.A.EFE.html download to ptt/Gossiping/20309_18.txt, time: 6.070000
/bbs/Gossiping/M.1495115665.A.F84.html download to ptt/Gossiping/20309_19.txt, time: 9.070000
page 20308
/bbs/Gossiping/M.1495114678.A.6F2.htmlTraceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 438, in send
    timeout=timeout
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 649, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
requests.packages.urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./crawler.py", line 177, in <module>
    main(sys.argv[1:])
  File "./crawler.py", line 158, in main
    downloadForum(forumName, startPage, pages)
  File "./crawler.py", line 140, in downloadForum
    downloadPage(str(articleCount), forumName, pageNum, 'https://www.ptt.cc'+link)
  File "./crawler.py", line 82, in downloadPage
    pageText = getPageText(link)
  File "./crawler.py", line 13, in getPageText
    pageText=requests.get(link, cookies={'over18':'1'}).text
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 518, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 639, in send
    r = adapter.send(request, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 488, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
